process/commands/CommandBuilder.java

```diff
@@ -0,0 +1,319 @@
+/*

```

Comment:
It seems that this file is already under `unitex/src/fr/umlv/unitex/process/commands/CommandBuilder.java` is there any reason to try to create a new copy under `process/commands/CommandBuilder.java`?